packages/selection/src/containers/SelectionContainer.spec.js

```diff
@@ -0,0 +1,493 @@
+      });
+    });
+
+    describe('onBlur', () => {
+      it('clears currently focused item', () => {
+        const firstItem = findItems(wrapper)
+          .first()
+          .simulate('focus');
+
+        expect(findItems(wrapper).first()).toHaveProp('data-focused', true);
+
+        firstItem.simulate('blur');
+        expect(findItems(wrapper).first()).toHaveProp('data-focused', false);
+      });
+    });
+
+    describe('onKeyDown', () => {
+      describe('ENTER keyCode', () => {
+        it('selects currently focused item', () => {
+          const firstItem = findItems(wrapper).first();
+
+          firstItem.simulate('focus');
+          firstItem.simulate('keydown', { keyCode: KEY_CODES.ENTER });
+          expect(findItems(wrapper).first()).toHaveProp('data-selected', true);
+        });
+      });
+
+      describe('SPACE keyCode', () => {
+        it('selects currently focused item', () => {
+          const firstItem = findItems(wrapper).first();
+
+          firstItem.simulate('focus');
+          firstItem.simulate('keydown', { keyCode: KEY_CODES.SPACE });
+          expect(findItems(wrapper).first()).toHaveProp('data-selected', true);
+        });
+      });
+
+      describe('HOME keyCode', () => {
+        it('focuses first available item', () => {
+          const lastItem = findItems(wrapper).last();
+
+          lastItem.simulate('focus');
+          lastItem.simulate('keydown', { keyCode: KEY_CODES.HOME });
+          expect(findItems(wrapper).first()).toHaveProp('data-focused', true);
+        });
+      });
+
+      describe('END keyCode', () => {
+        it('focuses last available item', () => {
+          const firstItem = findItems(wrapper).first();
+
+          firstItem.simulate('focus');
+          firstItem.simulate('keydown', { keyCode: KEY_CODES.END });
+          expect(findItems(wrapper).last()).toHaveProp('data-focused', true);
+        });
+      });
+
+      describe('while in horizontal mode', () => {
+        describe('LEFT keyCode', () => {
+          describe('when dir is LTR', () => {
+            it('decrements focusedIndex if currently greater than 0', () => {
+              const secondItem = findItems(wrapper).at(1);
+
+              secondItem.simulate('focus');
+              secondItem.simulate('keydown', { keyCode: KEY_CODES.LEFT });
+
+              expect(findItems(wrapper).first()).toHaveProp('data-focused', true);
+            });
+
+            it('decrements and wraps focusedIndex if currently less than or equal to 0', () => {
+              const firstItem = findItems(wrapper).first();
+
+              firstItem.simulate('focus');
+              firstItem.simulate('keydown', { keyCode: KEY_CODES.LEFT });
+              expect(findItems(wrapper).last()).toHaveProp('data-focused', true);
+            });
+          });
+
+          // TODO: enzyme doesn't yet support useContext
+          describe.skip('when dir is RTL', () => {
```

Comment:
as the comment says enzyme hasn't begun support for `useContext` yet so we can't test rtl support just yet https://github.com/airbnb/enzyme/issues/2011

Reply:
removed context usage in favour of dd12c5f so now we can easily test this